packages/ses/README.md

```diff
@@ -137,6 +146,61 @@ c1.globalThis === c2.globalThis; // false
 c1.globalThis.JSON === c2.globalThis.JSON; // true
 ```
 
+The global scope of every compartment includes a shallow, specialized copy of
+the JavaScript intrinsics, omitting `Date.now` and `Math.random` since these
+can be used as covert communication channels between programs that have not
+been expressly introduced.
+However, a compartment may be expressly given access to these objects
+through the first argument to the compartment constructor or by assigning
+them to the compartment's `globalThis` after construction.
+
+```js
+const powerfulCompartment = new Compartment({ Math });
+powerfulCompartment.globalThis.Date = Date;
+```
+
+### Compartment + Lockdown
+
+Compartment and Lockdown combine to isolate client code in an environment with
+limited powers and communication channels.
+A compartment receives only the capabilities it is expressly given and cannot
+modify any of the shared intrinsics.
+To do this, every compartment gets its own globals, including such
+objects as the `Function` constructor.
+Yet, Compartment and Lockdown do not break `instanceof` for any of these
+intrinsics types!
+
+All of the evaluators in one compartment are captured by that compartment's
+global scope, including `Function`, indirect `eval`, dynamic `import`, and its
+own `Compartment` constructor for child compartments.
+For example, the `Function` constructor in one compartment creates functions
+that evaluate in the global scope of that compartment.
+
+```js
+const f = new Function("return this");
+f() === globalThis
+// true
+```
+
+Lockdown prepares for compartments with separate globals by freezing
+their shared prototypes and replacing their prototype constructors
+with powerless dummies.
+So, `Function` is different in two compartments, `Function.prototype` is the
+same, and `Function` is not the same as `Function.prototype.constructor`.
+The `Function.prototype.constructor` can only throw exceptions.
+So, a function passed between compartments does not carry access to
+its compartment's globals along with it.
+Yet, `f instanceof Function` works, even when `f` and `Function` are
+from different compartments.
+
+The `globalThis` in each compartment is mutable.
+This can and should be frozen before running any dynamic code in that
+compartment, yet is not strictly necessary if the compartment only
+runs code from a single party.
```

Comment:
Suggest tightening to "It should be frozen before running any dynamic code in that compartment. However, if the compartment only runs code from a single party, freezing is not strictly necessary."  (also, better to put the if and condition before the associated then action).